biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -75,12 +82,41 @@ subroutine mortality_rates( cohort_in,bc_in,cmort,hmort,bmort,frmort )
     logical, parameter :: test_zero_mortality = .false. ! Developer test which
                                                         ! may help to debug carbon imbalances
                                                         ! and the like
+     
+   ! Size Dependent Senescence
+   ! rate (r) and inflection point (ip) define the increase in mortality rate with dbh
+    mort_r_size_senescence = EDPftvarcon_inst%mort_r_size_senescence(cohort_in%pft)
+    mort_ip_size_senescence = EDPftvarcon_inst%mort_ip_size_senescence(cohort_in%pft)
+
+       ! if param values have been set then calculate smort
+    if (mort_ip_size_senescence < 10000 ) then 
```

Comment:
change 10000 to 10000.0_r8?  or make some named constant?

Reply:
does FATES have a named constant for an arbitrary large number?

Reply:
there's this, but I'm not sure if it is the right really large number.
https://github.com/NGEET/fates/blob/master/main/FatesConstantsMod.F90#L136

Reply:
I'll change it to 10000.0-r8